src/test/java/com/prgrms/commandLineApplication/voucher/FixedAmountVoucherTest.java

```diff
@@ -15,17 +15,17 @@ class FixedAmountVoucherTest {
 
   @ParameterizedTest
   @CsvSource(value = {"fixed|3000|7000|10000", "fixed|5000|95000|100000", "fixed|1000|9000|10000"}, delimiter = '|')
-  @DisplayName("범위 내의 할인값을 입력했을 경우 Fixed Amount 계산 성공")
-  void fixedAmountVoucher_계산_성공(String voucherType, int discountAmount, int expected, int price) {
-    Voucher createdFixedVoucher = VoucherFactory.createVoucher(voucherType, discountAmount);
+  @DisplayName("범위 내의 할인값을 입력했을 경우 FixedAmount Discount 계산이 성공한다.")
+  void discountFixedAmount_Success(String voucherType, int discountAmount, int expected, int price) {
+    Voucher createdFixedVoucher = VoucherFactory.of(voucherType, discountAmount);
     int result = createdFixedVoucher.supplyDiscount(price);
     assertThat(result).isEqualTo(expected);
   }
 
   @ParameterizedTest
   @ValueSource(ints = {-1, -100, 200000})
-  @DisplayName("범위를 벗어난 할인값을 입력했을 경우 예외 발생 성공")
-  void fixedAmountVoucher_할인값_예외_발생(int discountAmount) {
+  @DisplayName("범위를 벗어난 할인값을 입력했을 경우 예외가 발생한다.")
+  void fixedAmountRange_ErrorSuccess(int discountAmount) {
```

Comment:
```suggestion
  void fixedAmountRange_ThrowException(int discountAmount) {
```
이런식이 낫지 않을까요?
에러와 성공 단어가 같이 있어서 어떤케이스를 테스트했는지 헷갈리네요